src/imagery/i.eodag/i.eodag.py

```diff
@@ -620,6 +654,91 @@ def products_compare(first, second):
     return search_result
 
 
+def save_footprints(search_result, map_name):
+    """Save products footprints as a vector map in the current mapset.
+
+    Reprojection is done on the fly.
+
+    :param search_result: EO products to be sorted
```

Comment:
Isn't this the search result per se, i.e., whatever is obtained from eodag after performing a search?

Reply:
Yup, copy-paste error. 😃